Patch-Source: https://github.com/theironrobin/siglo/pull/137
---
From 919fe1acfa71cc937d8d1e1bc52423611ffb5273 Mon Sep 17 00:00:00 2001
From: Achill Gilgenast <achill@achill.org>
Date: Sun, 16 Nov 2025 14:35:56 +0100
Subject: [PATCH] build: Install to /usr/lib/systemd/

Usually applications install into /usr, so that /etc/systemd will be
kept for system administrator overrides.
---
 data/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index 25d0c11..e28108c 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -18,7 +18,7 @@ install_data(join_paths('icons', 'com.github.theironrobin.siglo.svg'),
   install_dir: join_paths(get_option('datadir'), 'icons/hicolor/scalable/apps')
 )
 
-install_data('siglo.service', install_dir: '/etc/systemd/user/')
+install_data('siglo.service', install_dir: get_option('prefix') / 'lib' / 'systemd' / 'user')
 
 appstream_file = i18n.merge_file(
   input: 'com.github.theironrobin.siglo.appdata.xml.in',
